test/vulnz-check/mirror_images.py

```diff
@@ -0,0 +1,71 @@
+# %%
+from typing import NamedTuple
+import os
+
+import kfp
+from kfp import dsl
+from kfp.components import func_to_container_op, InputPath, OutputPath
+
+# %%
+# Mirror Image
+
+
+def mirror_image(
+        image: str,
+        source_registry: str,
+        destination_registry: str,
+        tag: str = '',
+):
```

Comment:
The component should probably output the destination_image, so that it can be chained.